pom.xml

```diff
@@ -51,6 +51,11 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.12.0</version>
+        </dependency>
```

Comment:
remove this dependency, it is already included in the spring starters

Reply:
i removed it but if you can check if what i will do in the util later to see any mistakes